<launch>
  <node pkg="fastsim" type="simu_fastsim" name="simu_fastsim" output="screen">
    <param name="settings" type="str" value="$(find hrl)/envs/labyrinthe.xml"/>
    <param name="path" type="str" value="$(find hrl)/envs" />
  </node>
</launch>